source/partials/release_notes/_release-19-9-0.md.erb

```diff
@@ -0,0 +1,69 @@
+<h4>Java 8 and 10 support removed</h4>
+
+Support for Java 8 and 10 has been dropped from this release. This is in line with the Java Licensing Changes we had announced earlier this year in this [blog post](https://www.gocd.org/2019/05/21/official-stance-on-java/).
+
+Starting 19.9.0, GoCD Server and Agent would run either on Java 11 or 12. Support for Java 13 will be introduced soon. GoCD installers from 19.6.0 come with Java Runtime pre-bundled, so users no longer need to install it separately.
```

Comment:
> GoCD Server and Agent would run either on Java 11 or 12

GoCD Server and Agent would only support Java 11 or 12

Reply:
Starting 19.9.0, GoCD Server and Agent ~~would~~ run/support ...

Reply:
done